src/test/java/uk/gov/hmcts/reform/prl/controllers/managedocuments/ManageDocumentsControllerTest.java

```diff
@@ -163,4 +164,19 @@ public void testHandleSubmitted() {
 
     }
 
+    @Test
+    public void testValidateCourtUserShouldReturnError() {
+        when(manageDocumentsService.checkIfUserIsCourtStaff(auth,callbackRequest)).thenReturn(false);
+        AboutToStartOrSubmitCallbackResponse response = manageDocumentsController.validateUserIfCourtSelected(auth, callbackRequest);
+        Assert.assertNotNull(response.getErrors());
+        Assert.assertTrue(!response.getErrors().isEmpty());
+        Assert.assertEquals("Only court admin/Judge can select the value 'court' for 'submitting on behalf of'", response.getErrors().get(0));
+    }
+
+    @Test
+    public void testValidateCourtUserShouldAllowToProcess() {
+        when(manageDocumentsService.checkIfUserIsCourtStaff(auth,callbackRequest)).thenReturn(true);
+        AboutToStartOrSubmitCallbackResponse response = manageDocumentsController.validateUserIfCourtSelected(auth, callbackRequest);
+        Assert.assertNotNull(response.getData());
```

Comment:
Please check for specific field rather than not null

Reply:
@shashisk97 added check for specific field